src/main/java/org/zalando/pazuzu/feature/Tag/TagDto.java

```diff
@@ -0,0 +1,32 @@
+package org.zalando.pazuzu.feature.tag;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Created by vpavlyshyn on 09/06/16.
+ */
+public class TagDto {
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @JsonProperty("name")
+    private String name;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/Pazuzu/pazuzu-registry/file/2756072084/issues/source?bid=3131965&fileBranchId=3355916#l18)